pkgs/tools/backup/luckybackup/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl
+, pkgconfig, libtool, qmake
+, rsync, ssh
+}:
+
+with stdenv.lib;
+stdenv.mkDerivation rec {
+
+  pname = "luckybackup";
+  version = "0.5.0";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/luckybackup/${version}/source/${pname}-${version}.tar.gz";
+    sha256 = "0nwjsk1j33pm8882jbj8h6nxn6n5ab9dxqpqkay65pfbhcjay0g8";
+  };
+
+  buildInputs = with stdenv.lib;
+    [ pkgconfig libtool rsync ssh ];
+
+  nativeBuildInputs = [ qmake ];
+  
+  prePatch = ''sed -i -e "s|/usr|$out|" luckybackup.pro'';
+
+  enableParallelBuilding = true;
+
+  meta = {
+    description = "A powerful, fast and reliable backup & sync tool";
+    longDescription = ''
+      luckyBackup is an application for data back-up and synchronization 
+      powered by the rsync tool.
+      
+      It is simple to use, fast (transfers over only changes made and not
+      all data), safe (keeps your data safe by checking all declared directories
+      before proceeding in any data manipulation ), reliable and fully 
+      customizable.
+    '';
+    homepage = http://luckybackup.sourceforge.net/;
```

Comment:
(nitpick) It's recommended to quote URLs now.

Reply:
Do you have a reference aboout that new issue? As far as I remember, the rationale was always unquote fully formed URLs (those without 'interpolation').

Reply:
https://github.com/NixOS/rfcs/pull/45

Reply:
Many thanks! I will update it accordingly.